src/modules/fancyzones/FancyZonesLib/Resources.resx

```diff
@@ -183,17 +183,17 @@
   <data name="Setting_Description_RestoreSize" xml:space="preserve">
     <value>Restore the original size of windows when unsnapping</value>
   </data>
-  <data name="Setting_Launch_Editor_Label" xml:space="preserve">
```

Comment:
These are internal labels which will retrigger localization. I don't think it's worth retriggering localization for all these strings just to change internal names.

Reply:
Hm, I see the point. So: revert all resw/rex files, right?

Reply:
LOL this entire file seems to be obsolete

Reply:
Yes, please revert the labels. Revert the changes from xaml / resource getting in code in case those are changed as well.

Reply:
Already done after your comment, check out the latest commits :)